site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -25,7 +25,7 @@ Also complete the "Enabling creation of RBAC resources" and "Installing Agones"
 ## Modify the code and push another new image
 
 ### Modify the simple-udp example source code
-Modify the main.go file. For example:
+Modify the [main.go](https://github.com/googleforgames/agones/blob/master/examples/simple-udp/main.go) file. For example:
```

Comment:
```suggestion
Modify the {{< ghlink href="examples/simple-udp/main.go" >}}main.go{{< /ghlink >}} file. For example:
```

Love it, but this shortcode will always link to appropriate release, rather than master (which may get broken at some point, because it's dev).

Otherwise, this is good to go!

Reply:
Applied your suggestion! Thanks!